webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -374,6 +374,10 @@ const ExperimentTrials: React.FC<Props> = ({ experiment }: Props) => {
     }
   }, [ fetchExperimentTrials, sendBatchActions ]);
 
+  const handleBatchAction = useCallback((action: string) => {
+    submitBatchAction(action as Action);
+  }, [ submitBatchAction ]);
+
```

Comment:
I'm curious why we're adding this function where it seems like it's only purpose is to cast action as an Action

Reply:
yeah this can probably just be an anon function